src/views/Layout.jsx

```diff
@@ -1,20 +1,29 @@
 import { Outlet, NavLink } from 'react-router-dom';
+import { IoHome, IoList, IoCreate } from 'react-icons/io5';
 
 import './Layout.css';
 import { auth } from '../api/config.js';
 import { useAuth, SignInButton, SignOutButton } from '../api/useAuth.jsx';
+import buddyLogo from '../../public/img/basket-buddy-logo.png';
 
 export function Layout() {
 	const { user } = useAuth();
+	const activeLink =
+		'Nav-link text-sage border-2 border-sage rounded-full bg-eggshell';
+	const link =
+		'Nav-link text-eggshell border-2 rounded-full border-vanilla  hover:text-sage hover:border-sage';
```

Comment:
Really like the navigation styling too!

Reply:
Same! Devina tracked down a good method to pass that active styling in, otherwise it was a bit tricky at first.